docs/build-ansible.rst

```diff
@@ -37,13 +37,24 @@ below.
 Building Ansible
 ~~~~~~~~~~~~~~~~
 
-Setup for the first alpha release
+Setup for building the first time
 `````````````````````````````````
 
+If you haven't built Ansible before, you'll need a checkout of the ansible-build-data repo.  This
```

Comment:
```suggestion
If you haven't built Ansible before, you'll need a checkout of the [ansible-build-data](https://github
com/ansible-community/ansible-build-data) repo.  This
```

Reply:
Tried that from my phone and it didn't work right, hopefully you can understand the intent :p